src/stylesheets/components/_np-score.scss

```diff
@@ -0,0 +1,76 @@
+.np-score {
+  margin-top: rem(45px);
+
+  text-align: center;
+
+  &__number-container {
+    width: fit-content;
+    height: rem(56px);
+    margin: 0 auto;
+    padding: 0;
+
+    list-style: none;
+
+    border: 0.5px solid map-get($brand-white, 'primary');
+    border-radius: rem(12px);
+  }
+
+  &__number-label {
+    display: inline-block;
+    height: rem(55px);
+    margin: 0;
+
+    border-right: 0.5px solid map-get($brand-white, 'primary');
+  }
+
+  &__number-label--no-border {
+    border-right: 0;
+  }
+
+  &__number {
+    margin: rem(15px 13px);
+
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'bolder');
+    font-size: rem(20px);
+    line-height: rem(25px);
+
+    cursor: pointer;
+    opacity: 0.5;
+
+    transition: color 200ms;
+  }
+
+  &__number-container:hover > &__number-label &__number {
+    opacity: 1;
+  }
+
+  &__number-container > &__number-label:hover ~ &__number-label &__number {
```

Comment:
<!--
  0 failure: 
  1 warning:  Expected ".np-sco...
  
  
  DangerID: danger-id-Danger;
  File: src/stylesheets/components/_np-score.scss;
  Line: 48;
-->

- :warning: Expected ".np-score__number-container > .np-score__number-label:hover ~ .np-score__number-label .np-score__number" to have no more than 3 compound selectors (selector-max-compound-selectors)